packages/image_picker/image_picker/example/ios/Podfile

```diff
@@ -31,7 +31,10 @@ target 'Runner' do
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
 
   target 'RunnerTests' do
+    platform :ios, '9.0'
```

Comment:
Why are you setting the tests to iOS 9? We still support iOS 8 for now. (The messages about deployment target when building with recent versions of Xcode should be harmless.)

Reply:
I set it because `OCMock` requires iOS 9.